sklearn/linear_model/tests/test_logistic.py

```diff
@@ -875,3 +875,11 @@ def test_warm_start():
                         assert_greater(2.0, cum_diff, msg)
                     else:
                         assert_greater(cum_diff, 2.0, msg)
+
+
+@ignore_warnings
```

Comment:
check seems to against auto only?

```
    if class_weight == "auto":
        class_weight_ = compute_class_weight(class_weight, mask_classes,
                                             y_bin)
        sample_weight *= class_weight_[le.fit_transform(y_bin)]
```
Do you mean some other line? sorry I don't get what you are saying!